docs/.vuepress/config.js

```diff
@@ -219,6 +219,14 @@ module.exports = {
             ['/guide/performance', 'Performance'],
           ]
         },
+        {
+          title: 'Upgrade and migration',
+          collapsable: false,
+          children: [
+            ['/guide/release-notes', 'Release notes'],
```

Comment:
Line 151 above: "['/guide/changelog', 'Changelog']," already has this

Reply:
@wojciechczerniak - done: 21f936a